js/main.js

```diff
@@ -1,5 +1 @@
-import {getRentalList} from './data.js';
-
-const REALTY_AMOUNT = 10;
-
-getRentalList(REALTY_AMOUNT);
+import './offers-details.js';
```

Comment:
такого модуля, `offers-details`, в списке файлов нет.

Reply:
```suggestion
import './popup.js';
```
Возможно это popup? или popup.js переименовать в `offers-details.js`